### PR TITLE
User auth

### PR DESCRIPTION
**Pull request by ShafinMahmudCS**
# Pull Request Description Template

Before making your pull request, these descriptions should be included to allow the reviewer to reference.
<ol>
    <li>What was the reason for the addition/changes made</li>
        <ul>
            <li>Ticket link (if there's no tickets, leave blank)</li>
            <li>Description of the reason for the changes: <b>Connected Frontend with the Backend through API calls</b></li>
        </ul>
    <li>Which directory can I find the changes in</li>
        <ul>
            <li>list the absolute path (within the repo) of the directories where the changed files are: <b>backend/src/main/java/com/MerchStore</b></li>
        </ul>
    <li>What other files or features are affected by the changes</li>
        <ul>
            <li>List the other features or functions that you intended to also connect or affect</li>
        </ul>
</ol>

### CODE REVIEW

File: frontend/src/components/Admin.js [hunk 1]
```diff
@@ -1,158 +1,255 @@
...
+          onClick={() => navigate("/")}
+        >
+          <ArrowBackIcon sx={{ fontSize: 40 }}></ArrowBackIcon>
+        </IconButton>{" "}
+        Back to Shop
+      </Typography>
+
+      <Box
+        className="container-admin"
+        sx={{
```
Comment (DROFEL) [inline frontend/src/components/Admin.js:100]: preferable way of styling would be to put these styles into separate object and then just use it where just like you would use css classes. But this still works

File: frontend/src/components/Merch.js [hunk 2]
```diff
@@ -117,25 +141,40 @@ export default function Merch() {
           </AppBar>
           <Box sx={centerItem}>
             <Typography variant="h4" display="inline" sx={[bold, fontIBM]}>
-              <Typography variant="h4" display="inline" sx={[bold, fontIBM, varColor]}>var </Typography> store = "
-              <Typography variant="h4" display="inline" sx={[bold, fontIBM, merchColor]}>MerchStore</Typography>";
+              <Typography
```
Comment (DROFEL) [inline frontend/src/components/Merch.js:144]: prettify noice👍

File: frontend/src/components/NavSidebar.js
Comment (DROFEL) [inline frontend/src/components/NavSidebar.js:1]: ive changed how navbar and sidebars are structured, but hopefully these changes wont interfere

**Review by DROFEL — APPROVED** [review]

File: frontend/src/components/Admin.js (hunk 1 above)
Comment (ShafinMahmudCS) [reply frontend/src/components/Admin.js:100]: @namanrai did the styling for this part.

**Pull request merged**